### ISSUE TITLE
add monthly payment parameter

### ISSUE THREAD

**Issue by simobenso** [jpederson/Accrue.js#9]
Hello, 

Would it be possible to add a monthly payment parameter as an option ?

**Comment by jpederson** [comment]
Well, it's sort of the purpose of the tool to *determine* the monthly payment and interest over the course of the loan based on term and amount. So if we're providing term and amount, why would we need to have monthly payment amount as well? Or do you mean you want to give the monthly payment and total amount, and then have it spit out the term? I guess I'm missing something, please elaborate on how you'd use this :) Not trying to be difficult.

**Comment by simobenso** [comment]
To be more specific, it would mean to add a new calculation method based on the rate,  the total amount and  a "desired monthly payment" and the response would generate the term among the other response values.

**Comment by jhaeger** [comment]
I think what simobenso is asking for is a "how much house can I afford" function. 

You populate the monthly payment, interest rate, but leave the "loan amount" field blank, which will be calculated based on the other fields.

**Comment by jpederson** [comment]
Absolutely, I'll look into building this out - I have this script in several production projects and would want to just make sure that any changes I make don't break the API. But I have this a few of the other issues in mind for future releases.
